charts/gha-runner-scale-set-controller/templates/deployment.yaml

```diff
@@ -54,10 +54,8 @@ spec:
         command:
         - "/manager"
         env:
-        - name: CONTROLLER_MANAGER_POD_NAME
-          valueFrom:
-            fieldRef:
-              fieldPath: metadata.name
+        - name: CONTROLLER_MANAGER_CONTAINER_IMAGE
+          value: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
         - name: CONTROLLER_MANAGER_POD_NAMESPACE
```

Comment:
do we want to change the `CONTROLLER_MANAGER_POD_NAMESPACE` to also read from `{{ .Release.Namespace }}`?

Reply:
It feels safer to me to reference a field if we can